net-im/qtox/qtox-1.15.0.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils unpacker xdg-utils
+
+DESCRIPTION="Most feature-rich GUI for net-libs/tox using Qt5"
+HOMEPAGE="https://github.com/qTox/qTox"
+SRC_URI="https://github.com/qTox/qTox/releases/download/v${PV}/v${PV}.tar.lz -> ${P}.tar.lz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk test X"
+
+S="${WORKDIR}"
```

Comment:
What a horrible packaging! Can you try using the GitHub snapshot instead? It would also kill the dep on lzip (→ unpacker) and give us sane directory layout.